sections/faqs/migration-v5.md

```diff
@@ -0,0 +1,22 @@
+## What do I need to do to migrate to v5?
+
+Ready for this?
+
+```sh
+npm install styled-components@beta react@^16.8 react-dom@^16.8 react-is@^16.8
+```
+
+_That's it._ 💥
+
+styled-components v5 does not introduce any breaking public API changes, and adds the following:
```

Comment:
Let's link to https://medium.com/styled-components/389747abd987, which should work even after publishing.